app/src/main/java/org/greenstand/android/TreeTracker/userselect/UserSelectScreen.kt

```diff
@@ -25,7 +25,9 @@ import org.greenstand.android.TreeTracker.view.LanguageButton
 import org.greenstand.android.TreeTracker.view.TextButton
 
 @Composable
-fun UserSelectScreen(viewModel: UserSelectViewModel = viewModel(factory = LocalViewModelFactory.current)) {
+fun UserSelectScreen(
+    viewModel: UserSelectViewModel = viewModel(factory = LocalViewModelFactory.current)
+) {
```

Comment:
There are tons of files that have nothing but formatting applied. Please remove all changes from files that didn't have real code changes.

Reply:
I'm not entirely sure why that happened. Super weird. I can definitely make sure that no useless updates are getting pushed through

Reply:
Thanks, and if you can't undo it, don't worry about it. After this PR I'd suggest looking into your IDE settings. I'm guessing the moment you open a file the IDE auto formats everything. This is nice if we have a unified formatter, but we don't. That's on the to-do list.